simcase-game/src/views/GameUI.js

```diff
@@ -2,22 +2,16 @@ import React from "react";
 import Header from "../components/Header";
 import ScenarioForm from "../components/ScenarioForm";
 import Ranking from "./Ranking";
+import { navigate } from "@reach/router";
 
 function GameUI({ id }) {
-  console.log(id);
-  const [scenario, setScenario] = React.useState(id);
-
-  function addScenario(station = null) {
-    setScenario(+scenario + 1);
-  }
-
   return (
     <div>
       <Header />
-      {scenario < 4 ? (
+      {id < 5 ? (
         <>
-          <h2>Scenario: {+scenario}</h2>
-          <ScenarioForm addScenario={addScenario} scenario={scenario} />
+          <h2>Scenario: {+id}</h2>
```

Comment:
Easy way to turn a `String` in to an `Integer`. One wonder if it is easier to read that one or `parseInt()`